config/jobs/kubernetes-sigs/aws-alb-ingress-controller/aws-alb-ingress-controller-presubmits.yaml

```diff
@@ -0,0 +1,28 @@
+presubmits:
+  kubernetes-sigs/aws-alb-ingress-controller:
+  - name: pull-aws-alb-ingress-controller-lint
+    always_run: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - lint
```

Comment:
Does this `make lint` pull down all the vendors for https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/master/go.mod? Think `kubekins-e2e` still uses Go 1.10?

Reply:
nice catch..it won't since we have migrated to go mod. Is there any go 1.11.1 kubekins-e2e version? otherwise, we have to make the vendor directory back.

Reply:
or we install go1.11.1 in make file if it doesn't exists 😸

Reply:
yeah a big :-1: to bringing vendor back..

Reply:
just checked it's already  using go1.11.1 in gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-master 😄